Addresses namespace issue with kubectl usage

DimmestP opened the pull request:
# EIDF Documentation Pull Request

## Description

Updates text in GPU Service Overview and FAQ in response to user queries:

### Attempts to access to GPU cluster without adding kubeconfig files

- Rewrote the service access section to try to highlight that it is the kubeconfig files that allows any VM within a project to access the service

### Confusion around total quota and long queue wait times

- Separated description of quota and job queueing by creating a new entry in the Project Queue section

### Users are submitting their requests to 'default' namespace not their project namespace

- New example use of kubectl in Overview page that explicitly uses -n flag
- Adds new section in FAQ giving an example forbidden error and the solution using the -n flag when using kubectl 
- Fixes #136 

## Type of change

- [ x ] Missing Documentation

## What has to be reviewed

Added new section to GPU Service FAQ:

 - Access to GPU Service resources in default namespace is 'Forbidden'

Rewrote sections in GPU Service Overview for clarity:

- Service Access
- Project Quotas
- Project Queues

## Checklist

- [ x ] Documentation follows the project style guidelines
- [ x ] Ensure Contact details contain Service Emails and Numbers
- [ x ] Self-review of documentation using mkdocs on local system
- [ x ] Spellcheck has been performed
- [ x ] Pre-commit has been run and passed

agngrant reviewed (changes requested):
Lines 39  - as the EIDF GPU Service is only accessible through EIDF Virtual Machines.

Line 41 - Existing Projects can request access to… through a Helpdesk request via the EIDF Portal (link to help form) or emailing <helpdesk>

Line 43 - New projects wanting to using the GPU Service should include this in their EIDF Project Application.

Line 47 - This namespace will normally be the EIDF Project code appended with ’ns’ … your example

Line 51 - the kubectl application

Line 81 - this need to be altered - the quota is not the sum of all requested resources - this isn’t clear what you mean here

Line 83 - this is only true for pods - Jobs will just be queued

DimmestP commented:
Removed line 81. I was trying to emphasise that resource requests can be of different sizes but the quota is over the whole project. I think people can get confused by asking for a job with 12 GPUs as that's in their quota.

Changed line 83 to refer to job requests rather than resource requests.

agngrant commented:
In line 15 of the FAQ - can the error message be fenced by a code block:
```bash
<content>
```
